pyrubberband/pyrb.py

```diff
@@ -66,8 +66,13 @@ def __rubberband(y, sr, **kwargs):
         subprocess.check_call(arguments)
 
         # Load the processed audio.
-        # Setting mono=False will ensure that the shape matches `y`
-        y_out, _ = librosa.load(outfile, sr=sr, mono=False)
+        y_out, _ = sf.read(outfile, always_2d=True)
+
+        # make sure that output dimensions matches input
+        if y.ndim == 1:
+            y_out = np.squeeze(y_out.T).T
```

Comment:
Is that last ``.T`` necessary?  If it's 1d (after squeeze) then a transpose should do nothing.